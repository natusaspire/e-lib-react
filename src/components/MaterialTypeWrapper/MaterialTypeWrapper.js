import React, { memo } from 'react';

const MaterialTypeWrapper = ({ children }) => (
  <div className="main">
    <div className="main__container main__detail-content">{children}</div>
  </div>
);

export default memo(MaterialTypeWrapper);
