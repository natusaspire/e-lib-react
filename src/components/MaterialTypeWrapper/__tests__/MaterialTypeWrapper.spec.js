import React from 'react';
import { shallow } from 'enzyme';

import MaterialTypeWrapper from '@/components/MaterialTypeWrapper';

describe('<MaterialTypeWrapper />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <MaterialTypeWrapper>
        <div>MaterialType</div>
      </MaterialTypeWrapper>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
