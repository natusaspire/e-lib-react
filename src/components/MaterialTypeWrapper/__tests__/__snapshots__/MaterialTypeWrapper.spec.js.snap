// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MaterialTypeWrapper /> should render correctly 1`] = `
<div
  className="main"
>
  <div
    className="main__container main__detail-content"
  >
    <div>
      MaterialType
    </div>
  </div>
</div>
`;
